src/react/actions/workflow.js

```diff
@@ -15,3 +20,69 @@ export function closeWorkflowEditNotification(): CloseWorkflowEditNotificationAc
         type: 'CLOSE_WORKFLOW_EDIT_NOTIFICATION',
     };
 }
+
+export function searchWorkflowsSuccess(workflows: APIWorkflows): SearchWorkflowsSuccessAction {
+    return {
+        type: 'SEARCH_WORKFLOWS_SUCCESS',
+        workflows,
+    };
+}
+
+// TO BE REMOVED BEFORE MERGE:
+// function MOCK_SEARCH_WORKFLOWS() {
+//     return Promise.resolve({
+//         body: [
+//             {
+//                 'replication': {
+//                     streamId: '123e4567-e89b-12d3-a456-426614174001',
+//                     name: 'replication1',
+//                     version: 1,
+//                     enabled: true,
+//                     source: {
+//                         prefix: 'myprefix',
+//                         bucketName: 'bucket0',
+//                     },
+//                     destination: {
+//                         locations: [{ name: 's3-loc' }],
+//                         preferredReadLocation: 's3-loc',
+//                     },
+//                 },
+//             },
+//             {
+//                 'replication': {
+//                     streamId: '123e4567-e89b-12d3-a456-426614174002',
+//                     name: 'replication2',
+//                     version: 1,
+//                     enabled: false,
+//                     source: {
+//                         prefix: 'myprefix2',
+//                         bucketName: 'bucket2',
+//                     },
+//                     destination: {
+//                         locations: [{ name: 's3-loc' }],
+//                         preferredReadLocation: 's3-loc',
+//                     },
+//                 },
+//             },
+//         ],
+//     });
+// }
```

Comment:
```suggestion
```

Reply:
Good catch, this one was here for mocking the API. Thank you :)